util/collate/unicode_ci_data.go

```diff
@@ -0,0 +1,394 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collate
+
+/* Created from allkeys.txt. Unicode version '4.0.0'. Do not EDIT */
```

Comment:
It's not from `allkeys.txt` directly.

Reply:
not from allkeys.txt directly, but data from allkeys.txt, updated